### DISCUSSION TITLE
miner_1  | /usr/local/bin/python: can't find '__main__' module in 'run.py'

### DISCUSSION THREAD

**Discussion by Mikyner** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#693] [Q&A]
`sudo docker-compose up

Creating network "twitch_default" with the default driver

Creating twitch_miner_1 ... done

Attaching to twitch_miner_1

miner_1  | /usr/local/bin/python: can't find '__main__' module in 'run.py'

twitch_miner_1 exited with code 1`

I have absolutely no idea where to find the problem.

**Comment by simonheros** [comment]
I was able to fix this issue on the Unraid Docker version. The "can't find '__main__' module in 'run.py'" is because it can't locate the run.py file.

For some reason on the Unraid docker, there's an folder named "run.py" nested with another empty folder named "run.py".

Despite the instructions suggesting: Container Path: /usr/src/app/run.py

I got it to work by placing the Python file inside the folder named "run.py".

/mnt/user/appdata/twitchminer/run.py/run.py
